libManager/libManager_test.go

```diff
@@ -0,0 +1,40 @@
+package libManager
+
+import (
+	"reflect"
+	"testing"
+
+	"github.com/future-architect/vuls/models"
+	godeptypes "github.com/knqyf263/go-dep-parser/pkg/types"
+)
+
+func TestScan(t *testing.T) {
+	var tests = []struct {
+		path   string
+		pkgs   []godeptypes.Library
+		expect []models.VulnInfo
+	}{
+		{
+			path: "app/package-lock.json",
+			pkgs: []godeptypes.Library{
+				{
+					Name:    "jquery",
+					Version: "2.2.4",
+				},
+				{
+					Name:    "@babel/traverse",
+					Version: "7.4.4",
+				},
+			},
+		},
+	}
+	for _, v := range tests {
+		actual, err := scan(v.path, v.pkgs)
+		if err != nil {
+			t.Errorf("error occured")
```

Comment:
__[Misspell]__ "occured" is a misspelling of "occurred" ([link](https://sider.review/gh/repos/54808920/pulls/829#issue-4176679))

<sub>posted by Sider</sub>